setup.cfg

```diff
@@ -80,6 +80,7 @@ test =
 docs =
     astropy>=5.0,<5.3
     numpy<2.0
+    sherpa<4.16
```

Comment:
See my comment above

Reply:
Thanks @bkhelifi . I have updated the motivation of the PR to explain why.